lib/CMakeLists.txt

```diff
@@ -14,16 +14,21 @@ set(LinkedLibs
   MLIRTosaDialect
   MLIRSupport
 
+  # Dialect passes.
   TorchMLIRTorchPasses
   TorchMLIRTorchConversionDialect
 
+  # Dialects.
```

Comment:
Both these should be dialects and the other one should be under `Dialect Passes.`